lib/main.coffee

```diff
@@ -46,7 +47,7 @@ module.exports =
         helpers.exec(@executablePath, args, {cwd: cwd}).then (output) ->
           return [] unless output?
           messages = []
-          @MessageRegexp ?= XRegExp regex, ''
+          @MessageRegexp = XRegExp regex, ''
```

Comment:
None.

For some bizarre reason the current file path is incorporated into the regex instead of filtered out in it. I'll cherry pick this into the branch and finish it up. Thanks @ndobson!